bin/prepare_multiqc.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import csv
+import argparse
+
+
+def find_file_in_subfolders(filename, root_folder='.'):
+    for root, dirs, files in os.walk(root_folder):
+        if filename in files:
+            return os.path.join(root, filename)
+    return None
```

Comment:
Introduced a separate function to look for a file within the subfolders in MultiQC, due to `stageAs: '?/*' in the staging strategy.